ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -505,9 +510,8 @@ registerGenesisStaking staking nes = nes {
     -- establish an initial stake distribution.
     initSnapShot :: SL.SnapShot (EraCrypto era)
     initSnapShot =
-        SL.stakeDistr
-          @era
-          (SL._utxo (SL._utxoState ledgerState))
+        SL.incrementalStakeDistr
```

Comment:
It's not obvious to me that this is correct. Maybe some comments re the `mempty` parts and the new use of the term "incremental"?

Reply:
The comments are good 👍 , but a git merge conflict was saved into the file.